core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -60,6 +68,7 @@ private User(UUID id, byte[] initializationVector, String firstName, String last
 			UserPrivate userPrivate, MessageDestination messageDestination)
 	{
 		super(id);
+		this.status = Status.UNCONFIRMED;
```

Comment:
Currently UNCONFIRMED users still can create buddy requests. This maybe should be disabled because it is a threat for buddies if someone signs up with a false identity and invites them?